pegasus_wrapper/workflow.py

```diff
@@ -861,6 +863,7 @@ def run_bash(
 
         commands_with_ckpt = list(command)
         commands_with_ckpt.append(f"touch {ckpt_path.absolute()}")
+        commands_with_ckpt.insert(0, f"cd {job_dir}")
```

Comment:
Can you explain what the bug was in the comments?

Reply:
Matching consistency with how the script which runs the Python jobs are generated. By default, the working directory is just a random directory where Pegasus is executing the job in but by moving the executing back to the location of the `Locator` for the job relative paths based on the Locators can work automatically without a user needing to explicitly convert to absolute paths.

Reply:
Awesome!